### PR TITLE
feat(COS-2506): added docs to Styles

### PR DESCRIPTION
**Pull request by kushki-dflasso**

### CODE REVIEW

**Review by LuisCuascota — APPROVED** [review]

File: src/module/Payment.ts [hunk 1]
```diff
@@ -204,16 +191,109 @@ import { ICard } from "repository/ICard.ts";
  * </body>
  * </html>
  * ```
- * ###### To Start with it necessary define style object
+ * ###### Definition Custom Styles
+ * If you want to apply custom styles to hosted files, Kushki SDK expose the interface {@link Styles}, so you have two ways to set your styles:
+ *  - Class Css.- The interface {@link CssProperties} allow received a string, You can put your class CSS
```
Comment (esteban-aristizabal) [inline src/module/Payment.ts:196]: Css Classes- The interface {@link CssProperties} allows to receive a string, so you can configure a CSS class of your site

File: src/module/Payment.ts [hunk 2]
```diff
@@ -204,16 +191,109 @@ import { ICard } from "repository/ICard.ts";
...
+ *  - Object [JSS](https://cssinjs.org/react-jss/?v=v10.3.0).- The interface {@link CssProperties} allow received an object, You can put your object with all CSS properties
```
Comment (esteban-aristizabal) [inline src/module/Payment.ts:197]: - JSS Object [JSS](https://cssinjs.org/react-jss/?v=v10.3.0).- The interface {@link CssProperties} allows to receive an object, so you can configure custom CSS styles

File: src/module/Payment.ts [hunk 3]
```diff
@@ -204,16 +191,109 @@ import { ICard } from "repository/ICard.ts";
...
+ *     fontFamily: "Arial,Verdana,Tahoma",
+ *     width: "300px"
+ *   },
+ *   focus: {
+ *     border: "1px solid #0077ff", //set styles in focus event to all inputs
+ *   }
+ *   cardNumber:  { //overwrite input styles
+ *     color "red",
+ *     width: "400px",
+ *     "&:focus": {
```
Comment (esteban-aristizabal) [inline src/module/Payment.ts:290]: "&:focus": { // this way you can configure styles for an specific field for the focus event

File: src/module/Payment.ts [hunk 4]
```diff
@@ -238,36 +318,25 @@ import { ICard } from "repository/ICard.ts";
...
- *      otp: {
+ *      otp: { //Add new attribute with otp field values
  *       inputType: "password",
  *       label: "OTP Verification",
  *       placeholder: "OTP Verification",
  *       selector: "id_otp"
  *     }
- *   }
+ *   },
+ *   styles: hostedFieldsStyles //Add new attribute with styles values
```
Comment (esteban-aristizabal) [inline src/module/Payment.ts:339]: // Add new attribute with styles values

**Review by esteban-aristizabal — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -125,21 +125,77 @@ const buildCardInstance = async () => {
   }
 }
 ```
-[More Examples](./wiki/Payment.md)
+[More Examples](./wiki/Payment.md#examples)
 
 ### &#xa0;&#xa0;&bull; Styling
+If you want to give custom styles to hosted files, Kushki SDK expose the interface [Styles](./wiki/interfaces/Payment.Styles.md), so you have two ways to set your styles:
```
Comment (esteban-aristizabal) [inline README.md]: If you want to give custom styles to hosted files, the interface [Styles](./wiki/interfaces/Payment.Styles.md) is exposed,

File: README.md [hunk 6]
```diff
@@ -125,21 +125,77 @@ const buildCardInstance = async () => {
...
+
+- Class Css.- The interface [CssProperties](./wiki/modules/Payment.md#cssproperties) allow received a string, You can put your class CSS
+- Object [JSS](https://cssinjs.org/react-jss/?v=v10.3.0).- The interface [CssProperties](./wiki/modules/Payment.md#cssproperties) allow received an object, You can put your object with all CSS properties
```
Comment (esteban-aristizabal) [inline README.md]: mismos comments de los JSDOCS

**Review by esteban-aristizabal — CHANGES_REQUESTED** [review]

**Review by esteban-aristizabal — APPROVED** [review]

**Pull request merged**